docs/overview/index.md

```diff
@@ -1,17 +1,105 @@
 ---
 title: Tezos overview
 last_update:
-  date: 11 September 2023
+  date: 15 August 2024
 ---
 
 Tezos is an open-source, decentralized blockchain [created in 2014](https://tezos.com/whitepaper.pdf) by Arthur and Kathleen Breitman. It raised awareness and support in 2017 with its crowdfunding and launched the following year. Since its inception, Tezos has gone through multiple iterations of upgrades and development, staying true to its ethos -- "a blockchain designed to evolve".
 
 Tezos has robust applications ranging from NFTs, DeFi, and gaming to enterprise and government use cases.
 
+## What is Tezos?
+
+At its core, Tezos is a worldwide network of computers called _nodes_, which anyone can run.
+The nodes all run the _Tezos protocol_, which is software that controls how the nodes communicate and agree on what to do.
+The result is a _decentralized_ computing platform that no one person or entity can control.
+
+## What can I do with Tezos?
+
+Developers can imagine Tezos as a computing platform that is spread across many computers worldwide and is therefore transparent, independent, fair, automated, and controlled by its users.
+They can do many of the same things with Tezos that they can with any other computing platform, but developers often use decentralized computing for these use cases:
+
+- **Cryptocurrencies**: Users can create and use digital currencies (also known as _tokens_) to use for digital currencies and to make it easier to accept online payments.
```

Comment:
something wrong in "create and use digital currencies ... to use for digital currencies"

Reply:
Thanks, updated to "Users can create and use digital objects known as _tokens_ to use as digital currencies and to make it easier to accept online payments."